app/models/concerns/scholars_archive/default_metadata.rb

```diff
@@ -184,6 +184,11 @@ module DefaultMetadata
         index.as :stored_searchable
       end
 
+      id_blank = proc { |attributes| attributes[:id].blank? }
+
+      class_attribute :controlled_properties
+      self.controlled_properties = [:based_near]
+      accepts_nested_attributes_for :based_near, reject_if: id_blank, allow_destroy: true
```

Comment:
I'd probably just put the proc directly in the attribute until/unless it is needed/used elsewhere